src/Fhir.Anonymizer.R4.ADF.UnitTests/Fhir.Anonymizer.R4.ADF.UnitTests.csproj

```diff
@@ -13,11 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Fhir.Anonymizer.AzureDataFactoryPipeline\Fhir.Anonymizer.AzureDataFactoryPipeline.csproj" />
-    <ProjectReference Include="..\Fhir.Anonymizer.Core\Fhir.Anonymizer.Core.csproj" />
-  </ItemGroup>
-
-  <ItemGroup>
+    <ProjectReference Include="..\Fhir.Anonymizer.R4.AzureDataFactoryPipeline\Fhir.Anonymizer.R4.AzureDataFactoryPipeline.csproj" />
```

Comment:
To keep ADF tool still working, we need to change the assembly name in Fhir.Anonymizer.R4.AzureDataFactoryPipeline/scripts/CustomActivity.ps1 to R4 version. And keep configuration.json.